app/src/pages/inside/productVersionPage/productVersionPage.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useMemo } from 'react';
+import { useIntl } from 'react-intl';
+import { useSelector } from 'react-redux';
+
+import { SettingsLayout } from 'layouts/settingsLayout';
+import { ScrollWrapper } from 'components/main/scrollWrapper';
+import { payloadSelector, urlOrganizationAndProjectSelector } from 'controllers/pages';
+import { PRODUCT_VERSION_TAB_PAGE } from 'controllers/pages/constants';
+import { ProductVersionHeader } from './productVersionHeader';
+import {
+  BUGS,
+  DOCUMENTATION,
+  HISTORY_OF_ACTIONS,
+  LINKED_TEST_CASES,
+  REQUIREMENTS,
+} from './constants';
+import { messages } from './messages';
+
+export const ProductVersionPage = () => {
+  const { formatMessage } = useIntl();
+  const { organizationSlug, projectSlug } = useSelector(urlOrganizationAndProjectSelector);
+  const { productVersionTab: currentTab, productVersionId } = useSelector(payloadSelector);
+
+  const createTabLink = useCallback(
+    (productVersionTab) => ({
+      type: PRODUCT_VERSION_TAB_PAGE,
+      payload: {
+        projectSlug,
+        organizationSlug,
+        productVersionId,
+        productVersionTab,
+      },
+    }),
+    [organizationSlug, productVersionId, projectSlug],
+  );
+
+  const tabsConfig = useMemo(
+    () => ({
+      [DOCUMENTATION]: {
+        name: formatMessage(messages.documentation),
+        link: createTabLink(DOCUMENTATION),
+        component: <div>{formatMessage(messages.documentation)}</div>,
```

Comment:
Do we really need empty `div`s here?

Reply:
not really, we will remove it during next stories, just wanted to show that tabs are working and rendering different content